code/__HELPERS/icons.dm

```diff
@@ -1051,13 +1051,12 @@ ColorTone(rgb, tone)
 	var/static/list/humanoid_icon_cache = list()
 	if(!icon_id || !humanoid_icon_cache[icon_id])
 		var/mob/living/carbon/human/dummy/body = generate_or_wait_for_human_dummy(dummy_key)
-
-		prefs?.copy_to(body,TRUE,FALSE)
-
-		J?.equip(body, TRUE, FALSE, outfit_override = outfit_override)
```

Comment:
ditto

Reply:
For the `else if`. There was a redundant equipping operation, else.